Open Judge System/Data/OJS.Data/Repositories/TestRepository.cs

```diff
@@ -9,15 +9,15 @@
 
     public class TestRepository : EfGenericRepository<Test>, ITestRepository
     {
-        public TestRepository(IOjsDbContext context)
+        public TestRepository(DbContext context)
             : base(context)
         {
         }
 
         public override void Delete(int id)
         {
             // TODO: Evaluate if this is the best solution
-            var test = this.Context.DbContext.ChangeTracker.Entries<Test>().FirstOrDefault(t => t.Property(pr => pr.Id).CurrentValue == id).Entity ?? new Test { Id = id };
+            var test = this.Context.ChangeTracker.Entries<Test>().FirstOrDefault(t => t.Property(pr => pr.Id).CurrentValue == id).Entity ?? new Test { Id = id };
```

Comment:
Shouldn't `.FirstOrDefault` and `.Entity` evaluation be on a new line? Also consider `.Entries`